Paws4Me/Paws4Me/Controllers/Logic/PetSingleDetailsViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  AdoptPet.swift
+//  Paws4Me
+//
+//  Created by Kirsty-lee Walker 20/02/2022
+//
+
+import UIKit
+
+class PetSingleDetailsViewController: UIViewController {
+    // MARK: - IBOutlets
+    @IBOutlet weak private var petImageView: UIImageView!
+    @IBOutlet weak private var petNameLabel: UILabel!
+    @IBOutlet weak private var petAgeLabel: UILabel!
+    @IBOutlet weak private var petGenderLabel: UILabel!
+    @IBOutlet weak private var petBreedNameLabel: UILabel!
+    @IBOutlet weak private var saveSinglePetButton: UIButton!
+
+    // MARK: - Vars/Lets
+    private var singlePet: AdoptPet?
+    private var indexSinglePet: Int = 0
+    private var namePet = ""
+    private var breedPet = ""
```

Comment:
`ViewController` does not need to know about all of these please pass them to the `ViewModel` and access them from there 😄